doc/tally-recovery.md

```diff
@@ -0,0 +1,82 @@
+# Recovery of the corrected vote tally from fragment logs
+
+In the Fund 4 time frame, we've been unable to completely resolve issues
+with the Catalyst app usage of spending counters in witnesses of successive
+ballot (aka VoteCast) transactions for the single voting account.
+This may lead to ballot transactions legitimately submitted by voting users
+to be rejected by the consensus leader nodes at block creation.
+
+To allow computation of the tally across all ballots legitimately submitted
+by the users in face of such discrepancies, an alternative
+tallying method was devised and implemented in the catalyst-toolbox tool.
+Afterwards, this method can be used to confirm the blockchain tally did not
+miss any received legitimate ballots that were recorded into persistent
+fragment logs.
+
+## Input: persistent fragment logs
```

Comment:
I think documenting the format or just adding the link to where it is defined would be good.